dspace-server-webapp/src/test/java/org/dspace/app/rest/VersionRestRepositoryIT.java

```diff
@@ -166,6 +175,10 @@ public void findOneForbiddenTest() throws Exception {
     @Test
     public void versionForItemTest() throws Exception {
 
+        context.turnOffAuthorisationSystem();
```

Comment:
we need to have test to check the access to an "inprogress" version that is the case (wrongly) checked by the previous version of this test. I'm ok to just note that in a separate ticket.